.github/CONTRIBUTING.md

```diff
@@ -80,34 +80,24 @@ Each contributor is required to [sign a CLA](https://cla.shopify.com/). This pro
 
 ### Contribution prerequisites
 
-- You have Node installed at v10.13.0+ and Yarn at v1.10.1+
+- You have Node installed at v16.13.0+ and Yarn at v1.22.18+
 
 ### Best practices
 
-Use the [documentation directory](https://github.com/Shopify/polaris-react/tree/main/documentation) for guidance on: [accessibility testing](https://github.com/Shopify/polaris-react/blob/main/documentation/Accessibility%20testing.md) and [adding to the change log](https://github.com/Shopify/polaris-react/blob/main/documentation/Versioning%20and%20changelog.md). For code style best practices, take a look at existing components.
+Use the [documentation directory](https://github.com/Shopify/polaris/tree/main/documentation) for guidance on: [accessibility testing](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md) and [adding to the changelog](https://github.com/Shopify/polaris/blob/main/documentation/Versioning%20and%20changelog.md). For code style best practices, take a look at existing components.
 
 Making sure the voice and tone is consistent across Polaris is important to us, for more information visit the [voice and tone guidelines](https://polaris.shopify.com/content/voice-and-tone).
 
 ### Development workflow
 
 After cloning Polaris React, run `yarn` to fetch its dependencies. Then you can run several commands:
 
-- `yarn dev` runs a Storybook server which includes a playground editable at `playground/Playground.tsx`
+- `yarn dev:react` runs a Storybook server which includes a playground editable at `playground/Playground.tsx`
 - `yarn test` runs the complete test suite
 - `yarn test <pattern>` runs tests with matching filenames
-- `yarn tophat` runs a local development server and mounts all of the component examples
 
 We recommend running `yarn test` (or its variations above), as well as trying your build of Polaris React in a real project, to make sure you don’t introduce any regressions as you work on your change.
 
-Run `yarn run build-consumer <project>` where `<project>` is the directory the build will be copied. **The `<project>` directory must be a sibling of the `polaris-react` directory**.
```

Comment:
Will create an issue to remove this `build-consumer` command as I don't know how much value it adds to our project.

Should we just remove it or would it be helpful to provide a `yalc`-based alternative for testing changes in a separate project? My thoughts are that it is out of scope for our documentation and recommendations.

@kyledurand @aveline

Reply:
I did just stumble across how this where the `build-consumer` command is used to test Component README changes in the `polaris-styleguide`.

Seems a bit unclear based on the steps how this all works and will likely change in the near future when we migrate documentation to the monorepo.

Are there alternate ways to test documentation updates you have tried (like `yalc`) or is this still the best solution?

Reply:
`build-consumer` is super useful for testing unreleased changes in web or also on the styleguide. I think it was added before yalc was a thing so updating it to use yalc would probably be the right move here.

Reply:
Agreed, +1 for updating the docs to demonstrate a `yalc` based workflow

Reply:
I've added some basic guidance on using `yalc` to test package changes, but would like to [do some more clean up](https://github.com/Shopify/polaris/issues/5550) and detailed tophatting instructions for `spin` as well.

I've broken that out into it's own task since it involves removing scripts and creating new doc files 📜➕➕➕